docs/paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "VirtualShip for simulating oceanographic fieldwork anywhere in the global ocean"
+tags:
+  - Python
+  - oceanography
+  - fieldwork simulation
+  - Lagrangian modelling
+  - data ingestion
```

Comment:
Not sure about this last tag. How about adding "(under)graduate training", to also keep the link with the educational side?